src/app/modules/dashboard/modules/day-plan/day-plan.component.scss

```diff
@@ -1,5 +1,11 @@
 .diet-day-plan-wrapper {
+  position: relative;
   height: 100%;
   background: url('~src/assets/images/day-plan-background.svg') no-repeat 50% 50%;
   background-size: contain;
+  overflow: auto;
+}
+
+.diet-day-plan-meal-list {
+  min-height: 50%;
```

Comment:
do czego to min-height?

Reply:
dopycha component ze statystykami do dołu jak jest mało posiłków